vNext/UnityGLTFv2.cs

```diff
@@ -66,7 +73,7 @@ public class UnityGLTFImporterConfig
 /// </summary>
 public class IDataLoader

```

Comment:
interface, not class
Also wondering about the name of this interface... I wonder if something like IResourceLocator would be more self explanatory? Not sure...

Reply:
It's not really locating resources. It's only resolving them to a stream.